libr/main/rafind2.c

```diff
@@ -498,44 +498,60 @@ R_API int r_main_rafind2(int argc, const char **argv) {
 			{
 				char *arg = strdup (opt.arg);
 				char *colon = strchr (arg, ':');
+				char *comma = NULL;
 				ut8 buf[8] = {0};
 				int size = (R_SYS_BITS & R_SYS_BITS_64)? 8: 4;
-				ut64 value = 0;
+				ut64 min_value = 0, max_value = 0;
+
 				if (colon) {
 					*colon++ = 0;
 					size = atoi (arg);
 					size = R_MIN (8, size);
 					size = R_MAX (1, size);
-					value = r_num_math (NULL, colon);
+					comma = strchr(colon, ',');
+
+					if (comma) {
+						*comma++ = 0;
+						min_value = r_num_math(NULL, colon);
```

Comment:
Use the proper indentation rules. Spaces before (, dont define variables in the for() block..

Reply:
@trufae I’ve made the requested changes, including updating the help message and ensuring proper indentation/styling. Please let me know if there’s anything else you’d like me to adjust.

Reply:
Perfect :) merged! Its always good to have some tests but not mandarory. Thank you for your first contribution!